FrontMen/fm-hours
# Feature/standby reports

# Changes
* Reports features an additional tab, for stand-by hours
* Additionally: refactor to use a generic method for adding up totals, based on an array of objects and a key in said object.

## Related issues
https://github.com/FrontMen/fm-hours/issues/189

## Added
Composable to get monthly totals for stand-by
Stand-by tab on /reports page

## Changed
Refactored: many places were using a fairly duplicated method for reducing an array and getting a total off of it. Made a generic helper and removed the duplicates

## How to test
1. Navigate to /reports
2. The totals should work as before (correct values)
3. Go to stand-by tab - a table with standby time for each employee should be present
4. Go to home
5. Go to monthly overview
6. Totals shown on page work as before (correct values)

## Screenshots
![image](https://user-images.githubusercontent.com/85111889/126679097-5532d0d1-e4d4-4484-9cb5-ba61db5aa9f2.png)
